twisted/conch/ssh/factory.py

```diff
@@ -58,7 +58,7 @@ def buildProtocol(self, addr):
                     'because we cannot find moduli file')
             t.supportedKeyExchanges = [
                 kexAlgorithm for kexAlgorithm in t.supportedKeyExchanges
-                if _kex.isFixedGroup(kexAlgorithm)]
+                if _kex.isFixedGroup(kexAlgorithm) or (hasattr(_kex, "isEllipticCurve") and _kex.isEllipticCurve(kexAlgorithm))]
```

Comment:
Is this `hasattr` necessary?  From what I can see, `isEllipticCurve` is not conditionally defined.